docs/application/dotnet/guides/applications/nui-widget-app.md

```diff
@@ -0,0 +1,243 @@
+# NUI Widget Application
+
+NUIWidgetApplication helps you to create light Tizen .NET applications on the home screen.
+
+**Figure: NUI widget application**
+
+![NUI widget application](./media/dali_widget_application.png)
+
+The main features of NUIWidgetApplication include:
+
+- Creating widget applications
+
+  You can [create a NUI widget application](#create-a-nui-widget-application), which usually is composed of one process having the main loop. The NUIWidgetApplication framework enables you to create a widget instance that shares the same resources within the process. The widget application can also share data with other applications.
+
+- Managing widget instances
+
+  Whenever a widget instance is requested, the framework creates one.
+  You can manage the widget instances by updating or destroying them, or by retrieving information about them.
+
+- Managing life-cycle
+
+  You can manage the [life-cycle of widget instances](widget-app.md#instance) through callback methods that are triggered as the instance states are changed.
+
+- Creating widget UI
+
+  The widget application can create a user interface that appears on the home screen or viewer.
+
+    > [!NOTE]
+    > The UI of widget application limits scroll actions to provide a better user experience. It is recommended that you design a UI layout within the given screen size.
+    >
+    > To draw the layout, you must use the [`Window`](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Window.html) instance received from the `OnCreate` callback. If you create additional windows, the stack of widget application windows gets corrupted. This is because the platform handles the widget application window in a particular way.
```

Comment:
```suggestion
    > To draw the layout, you must use the [`Window`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) instance received from the `OnCreate` callback. If you create additional windows, the stack of widget application windows gets corrupted. This is because the platform handles the widget application window in a particular way.
```